Adds docs for adding new users to backstage (#389)
RoadieHQ/marketing-site: content/docs/getting-started/user-management/index.md

---

> title: User Managment in Backstage

Reviewer: Changing to Roadie since this is not a Backstage feature.

```suggestion
title: User Management in Roadie
```

lastUpdated: '2021-09-21T14:00:00.0Z'

> description: How to manage users in Backstage

Reviewer: ```suggestion
description: How to manage users in Roadie.
```

---

## Introduction

> 

Reviewer 2: Might be worth adding the caveat that this only works if you are using auth0

Author: I put the caveat at the bottom of the page, so as not to interupt the flow, but I am also undecided. :/

Reviewer: Setup documentation is mostly used by people in free trials. All of the free trial people will be using Auth0. So most of the readers of this document will be using Auth0. For that reason, I like Brian's layout, with the comment about SSO at the bottom of the page.

Author: Alright cool. sure lets go with it the way it is.

Roadie provides backstage with an authentication and user management feature for your new tenant.

When the tenant is created the user requesting the tenant is added as the first user in backstage. This user may choose to add another user.

## User Management
The user management page is contained under the Administration side bar menu item. ![admin-sidebar.png](admin-sidebar.png)

> ![user-management-page.png](user-management-page.png)

Reviewer 2: Styling of these two look a bit off. Do you think we could just circle the gear on the second photo or something?

https://deploy-preview-389--roadie.netlify.app/docs/getting-started/user-management/

Author: Done

## Creating a User
To add a new user click the invite a new user button and enter the users email address.

![invite-user-button.png](invite-user-button.png)

Enter the users email and click send.

![add-user-dialog.png](add-user-dialog.png)

We will send an email to the users email address. The user will be asked to set a password and will them be redirected to login to backstage.

## Removing a User
At present we do not provide the ability to remove a user. However, we can take care of that on your behalf. Please speak to a support person and we will do that for you.

> ## Custom Authentication Provider

Reviewer: In my experience, most people call this "Single Sign On". Having that term in the heading will make this easier to find.

```suggestion
## Single Sign On
```

> Roadie supports Single Sign On via Okta or other providers. Please talk to the sales team if you would like to use Single Sign on

Reviewer: Sorry! I forgot a full stop.

```suggestion
Roadie supports Single Sign On via Okta or other providers. Please talk to the sales team if you would like to use Single Sign on.
```

> We can also configure your tenant to use your own authentication provider as a paid feature. Please talk to our sales team if this is what you would prefer.

Reviewer: ```suggestion
Roadie supports Single Sign On via Okta or other providers. Please talk to the sales team if you would like to use Single Sign on
```
